test/__snapshots__/index.js.snap

```diff
@@ -70,50 +70,7 @@ exports.resolvePages = () => {
 }"
 `;
 
-exports[`onBuild() generates static files manifest 1`] = `
-Array [
-  "en/getStaticProps/1.html",
-  "en/getStaticProps/1.json",
-  "en/getStaticProps/2.html",
-  "en/getStaticProps/2.json",
-  "en/getStaticProps/env.html",
-  "en/getStaticProps/env.json",
-  "en/getStaticProps/static.html",
-  "en/getStaticProps/static.json",
-  "en/getStaticProps/withFallback/3.html",
-  "en/getStaticProps/withFallback/3.json",
-  "en/getStaticProps/withFallback/4.html",
-  "en/getStaticProps/withFallback/4.json",
-  "en/getStaticProps/withFallback/my/path/1.html",
-  "en/getStaticProps/withFallback/my/path/1.json",
-  "en/getStaticProps/withFallback/my/path/2.html",
-  "en/getStaticProps/withFallback/my/path/2.json",
-  "en/getStaticProps/withFallbackBlocking/3.html",
-  "en/getStaticProps/withFallbackBlocking/3.json",
-  "en/getStaticProps/withFallbackBlocking/4.html",
-  "en/getStaticProps/withFallbackBlocking/4.json",
-  "en/image.html",
-  "en/previewTest.html",
-  "en/previewTest.json",
-  "en/static.html",
-  "es/getStaticProps/env.html",
-  "es/getStaticProps/env.json",
-  "es/getStaticProps/static.html",
-  "es/getStaticProps/static.json",
-  "es/image.html",
-  "es/previewTest.html",
-  "es/previewTest.json",
-  "es/static.html",
-  "fr/getStaticProps/env.html",
-  "fr/getStaticProps/env.json",
-  "fr/getStaticProps/static.html",
-  "fr/getStaticProps/static.json",
-  "fr/image.html",
-  "fr/previewTest.html",
-  "fr/previewTest.json",
-  "fr/static.html",
-]
-`;
+exports[`onBuild() generates static files manifest 1`] = `Array []`;
```

Comment:
You updated the snapshot after running `ntl build`. This is why I added the updateSnapshot npm script.